app/src/main/res/values-ar/strings.xml

```diff
@@ -401,7 +401,7 @@
   </plurals>
   <string name="topic_revision_recyclerview_tag">topic_revision_recyclerview_tag</string>
   <string name="ongoing_recycler_view_tag">ongoing_recycler_view_tag</string>
-  <string name="item_selection_text">برجاء اختيار خيار واحد على الأقل.</string>
+  <string name="item_selection_text">الرجاء تحديد جميع الخيارات الصحيحة.</string>
```

Comment:
For this & other languages, feel free to change them if you know the language otherwise you can leave the extra translations out since we have volunteers to ensure the correct translation is added for string changes.

Reply:
Hey @BenHenning Thanks i will revert changes to other languages string resources since i used google translate and i am not sure if it will 100% match the volunteers translation.